hw/top_earlgrey/cdc/cdc_waivers.tcl

```diff
@@ -50,6 +50,48 @@ set_rule_status -rule {W_GLITCH} -status {Waived}          \
     (GlitchOutput=~"*u_sync_1*")} \
   -comment {Waive PADs input goes into synchronizer}
 
+# Driving from PAD RESET to SPI TPM 
+set_rule_status -rule {W_ASYNC_RST_FLOPS} -status {Waived}                           \
+  -expression {(DrivingSignal=~"IO*") && (ReceivingFlop=~"*u_spi_device.*")} \
+  -comment {PAD RESET driving to SPI TPM}
+
+set_rule_status -rule {W_ASYNC_RST_FLOPS} -status {Waived}                           \
+  -expression {(DrivingSignal=~"SPI_DEV_CS_L*") && (ReceivingFlop=~"*u_spi_device.*")} \
+  -comment {PAD RESET driving to SPI TPM}
+
+set_rule_status -rule {W_ASYNC_RST_FLOPS} -status {Waived}                           \
+  -expression {(DrivingSignal=~"top_earlgrey.u_pinmux_aon*") && (ReceivingFlop=~"*u_spi_device.*")} \
+  -comment {PAD RESET driving to SPI TPM}
+
+set_rule_status -rule {W_ASYNC_RST_FLOPS} -status {Waived}                           \
+  -expression {(DrivingSignal=~"top_earlgrey.u_rstmgr_aon*") && (ReceivingFlop=~"*u_spi_device.*")} \
+  -comment {PAD RESET driving to SPI TPM}
+
+# Driving from PAD PIN MUX RESET to DMI JTAG 
+set_rule_status -rule {W_ASYNC_RST_FLOPS} -status {Waived}                           \
+  -expression {(DrivingSignal=~"top_earlgrey.u_pinmux_aon*") && (ReceivingFlop=~"*u_dmi_jtag.*")} \
+  -comment {PAD RESET driving to dmi jtag}
+
+# dual port memory to SPI
+set_rule_status -rule {W_MASYNC} -status {Waived}                           \
+  -expression {(Driver =~ "top_earlgrey.u_spi_device.u_memory_2p*") && (ReceivingFlop=~"top_earlgrey.u_spi_device.*")} \
```

Comment:
i'm slightly worried about this one. If hypothetically we hooked up thigns up wrong.. ie port A is clk, port B is spi_clk, but then we actually hook up some spi logic to read port A.

Not sure you could actually make this more specific..might not be possible.

Reply:
It's fixed to include more specific path. We are checking other issues related to dual port memory in parallel such as https://github.com/lowRISC/opentitan/issues/13834